cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js

```diff
@@ -50,6 +50,20 @@ function getPaymentCoreInfo(plan) {
     };
 }
 
+/**
+ * Returns the amount and rate of a credit properties for a given plan
+ * @param  {Object} plan any alma plan
+ * @returns {Object} with the properties translated
+ */
+function getCreditInfo(plan) {
+    var costOfCredit = parseInt(plan.customer_total_cost_amount / 100, 10).toString() + '.' + (plan.customer_total_cost_amount % 100) + '€';
```

Comment:
maybe use `Math.round()` when you ` / 100` ... no ?

Reply:
It's either `Math.round()` or `parseInt()`. round is more explicit to dev, parseInt is faster in execution.
I don't have a strong opinion on either one, if you prefer one over the other, tell me, I'll change it :)

Reply:
`Math.round()` is used everywhere else.
I prefer keep this more explicit method to to the job ;)